integration-tests/jpa-derby/.gitignore

```diff
@@ -0,0 +1 @@
+derby.log
```

Comment:
Please add it to the parent `.gitignore` (or even better if you can configure things to push the log to `target/`).